README.md

```diff
@@ -20,9 +20,9 @@ npm install deckardcain
 ## Basic Usage
 
 ```javascript
-const dc = require('deckardcain')
+import {identify} from '../lib/deckardcain';
```

Comment:
You're right if you're running the line from the library itself, because it is not in `node_modules` directory. Once it gets installed as a regular library, I guess it will work.